src/product_info/product_info.jsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState } from 'react';
+import './product_info.css'
+import { useParams } from 'react-router';
+import { Rating } from '@mui/material';
+const productPhotos = [
+    {
+        path: 'blue.svg',
+        name: 'blue'
+    },
+    {
+        path: 'grey.svg',
+        name: 'grey'
+    },
+    {
+        path: 'single_chair.svg',
+        name: 'single chair'
+    },
+    {
+        path: 'single_sofa.svg',
+        name: 'single sofa'
+    },
+    {
+        path: 'triple_chair.svg',
+        name: 'triple chair'
+    },
+]
+
+const messengers = [
+    {
+        path: 'facebook.svg',
+        name: 'Facebook'
+    },
+    {
+        path: 'linkedin.svg',
+        name: 'Linkedin'
+    },
+    {
+        path: 'twitter.svg',
+        name: 'Twitter'
+    },
+]
+
+const ProductInfo = () => {
+    const [productAmount, setProductAmount] = useState('1')
```

Comment:
use `number` type here
```suggestion
    const [productAmount, setProductAmount] = useState(1)
```